### DISCUSSION TITLE
Policy on sharing summary statistics

### DISCUSSION THREAD

**Discussion by kartikc727** [MIT-LCP/mimic-code#1628] [MIMIC-General] [answered]
Hi,

We are working on using MIMIC-III to create a Markov decision process (MDP) to model the evolution of the status of sepsis patients over time in response to different clinical interventions, similar to the [AI clinician paper](https://pubmed.ncbi.nlm.nih.gov/30349085/) that also uses MIMIC-III ([GitHub link](https://github.com/matthieukomorowski/AI_Clinician)).

The state of each patient at any given time step is clustered into `750` states, and two additional states corresponding to whether the patients survived or not, for a total of `752` states, and the clinical interventions are clustered into `25` actions. We then aggregate the number of transitions that take place from each initial state to each final state for each action and normalize them to create valid probability distributions in a `752 x 25 x 752` transition matrix. We similarly aggregate the number of times each state was seen as the first recorded state of a patient in the initial-state distribution, which is a vector of length `752`.

Our question is, can we make the resulting transition matrix open to the general public? This matrix only includes summary statistics about how patients falling into `752` different clusters tend to respond to treatments falling into `25` different clusters. Although this doesn't mean that we will be directly publishing MIMIC-III data, it would be summary statistics from that data, and so we wanted to ensure that posting our resulting MDP/transition matrix would not violate the Data Use Agreement (DUA), which says:

> I will not share access to PhysioNet restricted data with anyone else.

**Comment by alistairewj** [comment, marked as answer]
Always hard to draw the line between publishing findings and sharing the data, but this sounds like publishing a model from MIMIC, which has a lot of precedent. Just make sure there isn't some trivial way of regenerating individual level data (e.g. states of n=1).
